src/D2L.Security.OAuth2/packages.config

```diff
@@ -3,5 +3,7 @@
   <package id="D2L.Services.Core.Exceptions" version="1.0.0.27289" targetFramework="net451" />
   <package id="D2L.Services.Core.Extensions" version="1.1.1.0" targetFramework="net451" />
   <package id="Newtonsoft.Json" version="6.0.8" targetFramework="net451" />
+  <package id="System.Collections.Immutable" version="1.4.0" targetFramework="net461" />

```

Comment:
Mismatch with the LP, 

http://search.dev.d2l/source/search?&q=%22System.Collections.Immutable%22&path=packages.config&project=Lms&sort=fullpath

Reply:
Not sure if that's going to cause you troubles

Reply:
Interesting. I don't 100% remember what happens in the end either.

Any objections to updating LP?

Reply:
None from me.  That's at least the safer direction right?

Reply:
You want the later version of the DLL deployed, since it's a minor, other code should still be good.